articles/quickstart/spa/angular/01-login.md

```diff
@@ -158,6 +158,3 @@ The `user$` observable contains sensitive information and artifacts related to t
 Verify that you can display the `user.name` or [any other `user` property](/users/references/user-profile-structure#user-profile-attributes) within a component correctly after you have logged in.
 :::
 
-:::note
```

Comment:
@avanscoy The note is removed here but not added again in the file I think.

Reply:
I added the notes in the new style on the new commit. Sorry about that

Reply:
Thanks, Amanda. No worries!